### ISSUE TITLE
App name slightly confusing

### ISSUE THREAD

**Issue by schnillerman** [portapps/nextcloud-portable#55]
Hi, love your work!

Just stumbled over NC portable and was very curious because I thought it's a complete NC instance - portable?

But it looks like it's "only" the sync client. So as someone totally new to your apps - you might want to rename it to Nextcloud Sync Client or similar.

Just a remark, really.

**Comment by crazy-max** [comment]
Yes this is just the Nextcloud Desktop Client. I do agree this is confusing, I will look at changing its name.
